test/bdd/fixtures/discovery-config/sidetree-fabric/config-data/testnet.trustbloc.local.json

```diff
@@ -1,19 +0,0 @@
-{
-  "domain": "testnet.trustbloc.local",
```

Comment:
This domain is the stakeholder domain. Why deleted?

Reply:
we don't have load balancer locally we use peer container name as domain

Reply:
done